types/underscore/underscore-tests.ts

```diff
@@ -3529,6 +3439,127 @@ _.has({ a: 1, b: 2, c: 3 }, "b"); // $ExpectType boolean
     extractChainTypes(_.chain(anyValue).isUndefined()); // $ExpectType ChainType<boolean, never>
 }
 
+/***********
+ * Utility *
+ ***********/
+
+// noConflict
+
+// giving control of the _ global variable back to its previous owner (returns a reference to value of _ that is in effect before the function is called)
+_.noConflict(); // $ExpectType any
+
+// identity
+
+// calling a no-op function that returns the same value that is used as the argument
+_.identity({ name: 'moe' }); // $ExpectType { name: string; }
+
+// constant
+
+// creating a function that will always return a specific value
+_.constant({ name: 'moe' }); // $ExpectType () => { name: string; }
+
+// times
+
+// calling a function multiple times with the iteration as an argument and getting an array containing the result of each call
+// in this case, the result is the squares of the numbers 0 through 4
+_.times(5, n => n * n); // $ExpectType number[]
+
+// random
+
+// generating a random number between two bounds
+_.random(0, 100); // $ExpectType number
+
+// mixin
+
+// adding functions to Underscore by calling _.mixin and augmenting Underscore's
+// type definitions
+_.mixin({
+    capitalize: (string: string) => string.charAt(0).toUpperCase() + string.substring(1)
+});
+
```

Comment:
Works for me, the one additional change I'm going to make is to switch "TSC" to "TypeScript compiler" just to avoid possible confusion about the acronym.

Reply:
Actually, I think I'll drop the "compiler" part as well, this also helps IDEs provide autocomplete and possibly informs other helpful bits of the TypeScript ecosystem that may not be coming to my mind of this augmentation.